specs/core/0_beacon-chain.md

```diff
@@ -663,14 +662,13 @@ full_deposit_count: uint256
 @payable
 @public
 def deposit(deposit_input: bytes[2048]):
-    assert msg.value >= as_wei_value(MIN_DEPOSIT, "ether")
-    assert msg.value <= as_wei_value(MAX_DEPOSIT, "ether")
+    assert msg.value >= as_wei_value(MIN_DEPOSIT_AMOUNT, "gwei")
+    assert msg.value <= as_wei_value(MAX_DEPOSIT_AMOUNT, "gwei")
 
     index: uint256 = self.deposit_count + TWO_TO_POWER_OF_TREE_DEPTH
-    msg_gwei_bytes8: bytes[8] = slice(concat("", convert(msg.value / GWEI_PER_ETH, bytes32)), start=24, len=8)
-    timestamp_bytes8: bytes[8] = slice(concat("", convert(block.timestamp, bytes32)), start=24, len=8)
-    deposit_data: bytes[2064] = concat(msg_gwei_bytes8, timestamp_bytes8, deposit_input)
-    merkle_tree_index: bytes[8] = slice(concat("", convert(index, bytes32)), start=24, len=8)
+    amount: bytes[8] = slice(concat("", convert(msg.value / GWEI_PER_ETH, bytes32)), start=24, len=8)
+    timestamp: bytes[8] = slice(concat("", convert(block.timestamp, bytes32)), start=24, len=8)
```

Comment:
Since `timestamp` is a keyword for Vyper:
```
vyper.exceptions.VariableDeclarationException: Variable name invalid or reserved: timestamp
```
(Thanks @NIC619!)

Suggest keeping the old name: `timestamp_bytes8`

Reply:
What about `deposit_amount` and `deposit_timestamp`? Putting the type in the variable name feels wrong.